src/commands/check/index.ts

```diff
@@ -118,12 +153,27 @@ export default class Check extends SwankyCommand<typeof Check> {
           ctx.mismatchedVersions = mismatched;
         },
       },
+      {
+        title: "Check for missing tools",
```

Comment:
I uninstalled `cargo-dylint` to test this one. Here is my output:

```bash
➜  temp_project git:(main) ✗ ../bin/run.js check
✔ Check OS
✔ Check Rust
✔ Check cargo
✔ Check cargo nightly
✔ Check cargo dylint
✔ Check cargo-contract
✔ Check swanky node
✔ Read ink dependencies
✔ Verify ink version
✔ Check for missing tools

{ platform: 'darwin', architecture: 'arm64' } {
  tools: {
    rust: 'rustc 1.72.1 (d5c2e9c34 2023-09-13)',
    cargo: 'cargo 1.72.1 (103a7ff2e 2023-08-15)',
    cargoNightly: 'cargo 1.77.0-nightly (2ce45605d 2024-01-04)',
    cargoDylint: null,
    cargoContract: 'cargo-contract-contract 4.0.0-rc.1-unknown-aarch64-apple-darwin'
  },
  missingTools: [ 'cargoDylint' ],
  contracts: { flipper: { ink: '4.2.1' } },
  node: '1.6.0'
}
```

`cargoDylint` is correctly added to the missing tools, however `✔ Check cargo dylint` as a success tick. Use a "X" tick instead